index.html

```diff
@@ -8,310 +8,339 @@
+          </h3>
+          <ul class="list">
+            <li>Bootstrap</li>
+            <li>Ruby on Rails</li>
+            <li>Rspec</li>
+            <li>CapyBara</li>
+            <li>Selenium</li>
+          </ul>
+        </div>
+        <div class="all-about">
+          <h3 class="best">
+            <img src="icons/Rectangle57.png" alt="circle" /> Skills
+          </h3>
+          <ul class="list">
+            <li>Codekit</li>
+            <li>Github</li>
+            <li>Codepen</li>
+            <li>Gitlab</li>
+            <li>Terminal</li>
+          </ul>
+        </div>
+      </div>
+    </section>
+    <section class="contact-form">
+      <div class="the-contact">
+        <p class="contact-p">
+          I'm always interested in hearing about new projects, so if you'd like
+          to chat please get in touch.
+        </p>
+
+        <form action="https://formspree.io/f/xnqwjqwy" method="post">
+          <div class="input__sec">
+            <div class="name">
+              <label>
+                <input
+                  type="text"
+                  id="full-name"
+                  name="name"
+                  placeholder="Full Name"
+                  required
+                  minlength="2"
+                  maxlength="30"
+                />
+              </label>
+            </div>
+            <p>
+              <label>
+                <input
+                  type="email"
+                  name="email"
+                  placeholder="Email address"
+                  required
+                />
+              </label>
+            </p>
+            <label>
+              <textarea
+                name="textarea"
                 required
-                maxlength="30"
-                placeholder="Your Email"
-            /></label>
-            <textarea
-              id="message"
-              cols="30"
-              rows="10"
-              maxlength="500"
-              placeholder="Write your message here"
-              required
-            ></textarea>
-            <div class="submit-button">
-              <button type="submit">Get in touch</button>
+                minlength="2"
+                maxlength="500"
+                placeholder="Hello Vistor! Glad to see you!"
+              ></textarea>
+            </label>
+            <div class="btn__container">
+              <button type="submit" name="submit" class="btn__submit">
+                Get in touch
+              </button>
             </div>
-          </form>
-        </div>
-      </section>
-    </main>
+          </div>
+        </form>
+      </div>
+      <div class="hr1">
+        <hr class="hr" />
+      </div>
+      <div class="last-contact">
+        <ul class="footer-contacts">
+          <li>
+            <a href="https://github.com/mesfin345"
+              ><img src="icons/github.svg" alt="github addres"
+            /></a>
+          </li>
```

Comment:
[OPTIONAL] If you couldn't change the color of the social icons, Please look for new icons with the same color that is provided in the Figma template and use them in the contact section. You can check out this website for that: [Fontawesome](https://fontawesome.com/)